sections/05_treat.md

```diff
@@ -257,11 +257,11 @@ study of BACE-1 inhibitors included only 1547 compounds
 this limited dataset, but overfitting was a challenge and the differences
 between random forests and a deep neural were negligible, especially in the
 classification setting.  Overfitting is still a problem in larger chemical
-screening datasets with tens or hundreds of thousands of compounds because the
-number of active compounds can be very small, on the order of 0.1% or 1% of all
-tested chemicals for a typical target  `TODO: verify those estimates`. This is
-consistent with the strong performance of low-parameter neural networks that
-emphasize compound-compound similarity, such as influence-relevance voter,
+screening datasets with tens or hundreds of thousands of compounds.  For a
```

Comment:
I think in the previous version it was perhaps clearer why overfitting is still problematic. Now you have two sentences that may or may not be related.